examples/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+project(examples)
+
+cmake_minimum_required(VERSION 3.2)
```

Comment:
Are we sure about this version?

Reply:
Not at all, do you want to find out the min version ?

Reply:
Not necessary in this PR, but it would be great to find out what version we are really using.

Reply:
done in this PR, libf3d is cmake 3.1 (or earlier, did not check cmake 2), plugins and F3D are 3.13. (` Unknown CMake command "target_link_options".`